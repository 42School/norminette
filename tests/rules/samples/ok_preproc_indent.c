#define STUFF 1
#ifndef FOO
# define FOO
# ifndef BAR
#  define BAR
# endif
#endif
