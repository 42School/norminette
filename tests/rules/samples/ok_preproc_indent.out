[36mok_preproc_indent.c[0m - [32mIsPreprocessorStatement[0m In "GlobalScope" from "None" line 1":
		<DEFINE=#define STUFF 1> <NEWLINE>
[36mok_preproc_indent.c[0m - [32mIsPreprocessorStatement[0m In "GlobalScope" from "None" line 2":
		<IFNDEF=#ifndef FOO> <NEWLINE>
[36mok_preproc_indent.c[0m - [32mIsPreprocessorStatement[0m In "GlobalScope" from "None" line 3":
		<DEFINE=# define FOO> <NEWLINE>
[36mok_preproc_indent.c[0m - [32mIsPreprocessorStatement[0m In "GlobalScope" from "None" line 4":
		<IFNDEF=# ifndef BAR> <NEWLINE>
[36mok_preproc_indent.c[0m - [32mIsPreprocessorStatement[0m In "GlobalScope" from "None" line 5":
		<DEFINE=#  define BAR> <NEWLINE>
[36mok_preproc_indent.c[0m - [32mIsPreprocessorStatement[0m In "GlobalScope" from "None" line 6":
		<ENDIF=# endif> <NEWLINE>
[36mok_preproc_indent.c[0m - [32mIsPreprocessorStatement[0m In "GlobalScope" from "None" line 7":
		<ENDIF=#endif> <NEWLINE>
ok_preproc_indent.c: Error!
Error: INVALID_HEADER       (line:   1, col:   1):	Missing or invalid 42 header
